packages/dev/parcel-namer-s2-icon/package.json

```diff
@@ -1,12 +1,14 @@
 {
-  "name": "parcel-namer-icon",
+  "name": "@react-spectrum/parcel-namer-s2-icon",
   "version": "1.0.0",
-  "private": true,
   "main": "IconNamer.js",
   "engines": {
     "parcel": "2.0.0-dev.1599"
   },
   "dependencies": {
     "@parcel/plugin": "2.0.0-dev.1601"
+  },
+  "rsp": {
+    "type": "cli"
   }
```

Comment:
These will need `publishConfig` as well. Not sure why the linter isn't catching those

Reply:
done, updated the constraints to catch it as well
my categories aren't quite right https://github.com/adobe/react-spectrum/pull/6806/files#diff-594447fef9feecf11f1839e337c464b7b857fdb73fde7e18940eaa9cdf5b819dR165
I'd like to change this to just check for `private` but then I fail a bunch of other packages

those other packages should have some sort of category, like, the types packages don't need swc-helper or CSS side effects, but I haven't figured out what those categories are yet
icons packages also fit into the not needing CSS side effects constraint

Reply:
Bit confused here, so do these packages need the `publishConfig` or not? I assumed yes because otherwise it would try to publish them as a private package by default which needs a paid npm account (at least thats what I ran into when fixing the nightlies) and the changes to the linter seem to indicate that these cli packages should still be included if I'm reading the changes correctly but the package.jsons don't have `publishConfig` with public access

Reply:
yes, all packages that we publish need a publishConfig